NohBoard/Forms/MainForm.Editmode.cs

```diff
@@ -703,6 +703,10 @@ void OnDefinitionChanged(ElementDefinition def)
             {
                 this.elementUnderCursor = null;
                 this.currentlyManipulating = null;
+                if (def.Equals(this.selectedDefinition))
```

Comment:
This should always be the case, as long as `this.selectedDefinition` is not `null`, right? I mean, if something is selected, and `def` is something else, then there's something going seriously wrong.

Still, hypothetically, if that were the case, apparently we're not editing the selected definition, so we should probably leave it alone. So I'll go with your implementation.

Reply:
You can open the "Element Properties" dialog without selecting a key first. That's why I assumed that key selection and the dialog are not coupled and I thought that you could select one key and edit another, which is apparently not the case as I just found out. So you're right. A null check would be sufficient here.

I could update the PR but as you said, it probably won't hurt to leave it as it is.